src/libraries/Microsoft.Extensions.Http/tests/Microsoft.Extensions.Http.Tests/Logging/RedactedLogValueIntegrationTest.cs

```diff
@@ -16,6 +16,15 @@ namespace Microsoft.Extensions.Http.Logging
 {
     public class RedactedLogValueIntegrationTest
     {
+        private static class EventIds
```

Comment:
I think we should be able to access LogHelper from the tests -- we have InternalsVisibleTo set up here https://github.com/dotnet/runtime/blob/main/src/libraries/Microsoft.Extensions.Http/src/Properties/InternalsVisibleTo.cs#L6

Reply:
I'm not big fan of reusing product logic in it's own tests, it might hurt some aspects of the testing. Eg. if you accidentally change the id in the test will not detect the breaking change. Or is there another reason for the dependency I'm missing?